src/Tizen.NUI.Scene3D/src/public/ModelMotion/MotionValue.cs

```diff
@@ -33,38 +33,38 @@ namespace Tizen.NUI.Scene3D
     /// <remarks>
     /// The type of property should be matched with MotionIndex required.
     /// </remarks>
-    [EditorBrowsable(EditorBrowsableState.Never)]
+    /// <since_tizen> 11 </since_tizen>
     public class MotionValue : BaseHandle
     {
         /// <summary>
         /// Determine whether current stored value is PropertyValue, or KeyFrames.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 11 </since_tizen>
         public enum ValueType
         {
             /// <summary>
             /// Value is null, or invalid class.
             /// </summary>
-            [EditorBrowsable(EditorBrowsableState.Never)]
+            /// <since_tizen> 11 </since_tizen>
             Invalid = -1,
 
             /// <summary>
             /// Value is PropertyValue.
             /// </summary>
-            [EditorBrowsable(EditorBrowsableState.Never)]
+            /// <since_tizen> 11 </since_tizen>
             Property = 0,
 
             /// <summary>
             /// Value is KeyFrames.
             /// </summary>
-            [EditorBrowsable(EditorBrowsableState.Never)]
+            /// <since_tizen> 11 </since_tizen>
             KeyFrames,
         }
 
         /// <summary>
         /// Create an initialized motion value with invalid.
```

Comment:
"motion value with invalid" -> "invalid/empty motion value", "motion value with invalid sth...", "motion value with <cref name="Type"/> not set/initialized"

Reply:
I make it as invalid motion value.